scripts/algorithms/l2-svm.dml

```diff
@@ -144,10 +144,6 @@ while(continue == 1 & iter < maxiterations)  {
     s = be * s + g_new
     g_old = g_new
 
-	if(sum(s^2) == 0){
-	    continue = 0
-	}
```

Comment:
that is fine - thanks @iyounus (I'll make a detailed pass later tonight); removing the branches for continue, should be as simple as "continue1 = (g*g/h >= 0.0000000001);" and "continue = (step_sz*tmp >= epsilon*obj & sum(s^2) != 0);", but I can take care of that later as well.